src/databricks/labs/ucx/install.py

```diff
@@ -962,6 +963,21 @@ def validate_and_run(self, step: str):
         if not self.validate_step(step):
             self.run_workflow(step)
 
+    def _enable_files_in_repos(self):
+        # check if "enableProjectTypeInWorkspace" and "enableWorkspaceFilesystem" are set to false
+        project_type = self._ws.workspace_conf.get_status("enableProjectTypeInWorkspace")
+        workspace_file_system = self._ws.workspace_conf.get_status("enableWorkspaceFilesystem")
+
+        logger.debug("Checking Files In Repos configuration")
+
+        if project_type["enableProjectTypeInWorkspace"] == "false":
+            logger.debug("enableProjectTypeInWorkspace is False, enabling the config")
+            self._ws.workspace_conf.set_status("enableProjectTypeInWorkspace")
+
+        if workspace_file_system["enableWorkspaceFilesystem"] == "false":
+            logger.debug("enableWorkspaceFilesystem is False, enabling the config")
+            self._ws.workspace_conf.set_status("enableWorkspaceFilesystem")
```

Comment:
Method expects you to have dictionary https://databricks-sdk-py.readthedocs.io/en/latest/workspace/settings/workspace_conf.html#databricks.sdk.service.settings.WorkspaceConfAPI.set_status